code-of-conduct.md

```diff
@@ -0,0 +1,64 @@
+
+In the interest of fostering an open and welcoming environment in the OHBM Open
+Science Demo Calls, we want participation in our project and our community to be
+a harassment-free experience for everyone.
+We therefore respect and promote diversity in all of our spaces, including the
+GitHub repository, online calls, and accompanying live chat.
+
+Although no list can hope to be all-encompassing, we explicitly honor diversity in age,
+body size, disability, ethnicity, gender identity and expression, level of experience, native language, education, socio-economic status, nationality, personal appearance, race, religion,
+or sexual identity and orientation.
+
+## Our Standards
+
+We aim to promote behavior that contributes to a positive and welcoming environment.
+Examples of such behavior include:
+
+* Using inclusive language
+* Being respectful of differing viewpoints and experiences
+* Showing empathy towards other community members
+
+We do not tolerate harassment or other, inappropriate behavior in our community spaces.
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+  advances
+* Insulting/derogatory comments about a project or its maintainer(s), and
+  personal or political attacks
+* Public or private harassment
+* Repeated off-topic remarks with an intent to disrupt or derail ongoing
+  conversation
+
+## Our Responsibilities
+
+The host and appointed moderators are responsible for clarifying the standards
+of acceptable behavior and are expected to take appropriate and fair corrective
+action in response to any instances of unacceptable behavior.
+
+The host and appointed moderators have the right and responsibility to remove,
+edit, or reject comments, commits, code, issues, and other contributions
+that are not aligned to this Code of Conduct, or to ban temporarily or
+permanently any contributor for other behaviors that they deem inappropriate,
+threatening, offensive, or harmful.
+
+## Scope
+
+This Code of Conduct applies both within OHBM Open Science Demo Calls online spaces
+and in public spaces when an individual is representing the project or its community.
+Examples of representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+Representation of a project may be further defined and clarified by project maintainers.
+
```

Comment:
I'd recommend deleting the "Representation of a project may be further defined and clarified by project maintainers." sentence - I think that's a placeholder.

I think the examples don't really fit our use. How about:

"An example of representing the OHBM Open Science demo calls would be giving a talk at an event about the project."